homeassistant/components/goalfeed.py

```diff
@@ -0,0 +1,60 @@
+"""Component for the Goalfeed service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/goalfeed/
+"""
+import json
+
+import requests
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+DOMAIN = 'goalfeed'
+
+REQUIREMENTS = ['pysher==0.2.0']
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+GOALFEED_HOST = 'feed.goalfeed.ca'
+GOALFEED_AUTH_ENDPOINT = 'https://goalfeed.ca/feed/auth'
+GOALFEED_APP_ID = 'bfd4ed98c1ff22c04074'
```

Comment:
Is this a generic ID?

Reply:
Yep, actually this id is common for all users and pertains to the data source